index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Front-End School</title>
+  <link rel="stylesheet" href="./src/styles.css">
+</head>
+
+<body>
+  <!-- A DIV which contains header & headline background image -->
+  <div class="headline-bg">
+    <header>
+      <div>
+        <ul class="top-header-list">
+          <li><img src="./Images/facebook.png" alt="facebook icon"></li>
+          <li><img src="./Images/twitter.png" alt="twitter icon"></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li class="last"><a href="#">Log out</a></li>
+        </ul>
+      </div>
+
+      <nav>
+        <!-- Desktop Navbar -->
+        <div class="desktop-nav">
+          <img class="page-logo" src="./Images/logo.png" alt="page logo">
+          <ul class="desktop-nav-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#services">Our Services</a></li>
+            <li><a href="#team-container">Our Team</a></li>
+            <li><a href="#supporters">Sponsors</a></li>
+            <li class="last"><a class="list-btn" href="#">Our Projects</a></li>
+          </ul>
+        </div>
+
+        <!-- Mobile Navbar -->
+        <div>
+          <img id="hamburger-icon" class="union-icon" src="./Images/hamburger.png" alt="hamburger icon">
+
+          <!-- close icon-->
+          <a href="#" class="close-icon hidden">
+            <img src="./Images/close.png" alt="close icon">
+          </a>
+
+          <!-- mobile menu -->
+          <ul class="mobile-menu hidden">
+            <li class="mobile-list"><a href="./about.html">About</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#services">Our Services</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#team-container">Our Team</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#supporters">Sponsors</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#">Our Projects</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <article class="healine-title">
+        <p class="to-viewers">"Hello, front-end lovers!"</p>
+        <h1 class="frontend">Front-End School, best place to learn frontend technologies!</h1>
+      </article>
```

Comment:
The text size looks a bit larger on the mobile screen, kindly make sure to make it similar to the original design.

Yours:
![image](https://user-images.githubusercontent.com/34779928/205142744-84554582-26c8-4e75-baf6-93174d6fe630.png)

Original:
![image](https://user-images.githubusercontent.com/34779928/205142796-67d9f6a9-d6dc-49fc-bcf9-8d3678043ceb.png)